cmd/mountlib/rc.go

```diff
@@ -4,29 +4,40 @@ import (
 	"context"
 	"log"
 	"sort"
+	"sync"
+	"time"
 
 	"github.com/pkg/errors"
 	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/rc"
 )
 
+// MountInfo defines the configuration for a mount
+type MountInfo struct {
+	unmountFn  UnmountFn
+	MountPoint string    `json:"MountPoint"`
+	MountedOn  time.Time `json:"MountedOn"`
+	Fs         string    `json:"Fs"`
+}
+
 var (
 	// Mount functions available
-	mountFns map[string]MountFn
-	// Map of mounted path => unmount function
-	unmountFns map[string]UnmountFn
+	mountFns = map[string]MountFn{}
```

Comment:
I would just have 1 mutex to protect both maps otherwise you'll get confused about which one you've got locked!

It is traditional in go code to call them `mu` or `somethingMu` so lets say it is called `mountsMu`.

It is also traditional to define them **before** the things they protect, so I think this block should look like

```go
    // mutex to protect all the variables in this block
    mountMu sync.Mutex
    // Mount functions available
    mountFns = map[string]MountFn{}
    // Map of mounted path => MountInfo
    liveMounts = map[string]MountInfo{}
```